### ISSUE TITLE
Task: Add search engine meta tags for the site

### ISSUE THREAD

**Issue by tamalchowdhury** [tamalchowdhury/bankcardsbangladesh#21]
Hello dev, please add some meta tags for the site. You can add this from the editor or directly from the GitHub UI.

How to add meta tags:

Edit the `index.html` and edit the `<title>` tag:

```
<title>Bank Card Bangladesh</title>
```

You can write an SEO title up to 70 characters. So feel free to extend this title.

Example: `Bank Card Bangladesh | Find all debit, credit card information and fees`

Then add a meta description tag, after the title tag:

```
<meta name="description" content="[site description tag goes here...]">
```

Example:

```
<meta name="description" content="Upsetting the small town of Whoville, a local elderly man steals everyone's presents the night before an important event. Stay tuned for live updates on the matter.">
```

Here, come up with a 150-character site description.

You can use chat GPT to help you write a meta site description. [chat gpt](https://chat.openai.com/)

All the best and thanks a lot for your contribution.

**Comment by tamalchowdhury** [comment]
@farhanajuli @Pritam-2002 anyone wanna take this? This will help you complete your Hacktoberfest quota I hope.

**Comment by Malay-dev** [comment]
Hey @tamalchowdhury, Can I take up this issue...

**Comment by ankman007** [comment]
@tamalchowdhury Working on this issue. Will make PR shortly. I have experience with SEO, so coming up with meta title & description should not be an problem.

**Comment by tamalchowdhury** [comment]
@ankman007 I have assigned this to you. I will review your PR.

**Comment by tamalchowdhury** [comment]
@Malay-dev it's already taken. How about adding open graph tags for Facebook and Twitter? Do you know how to add these tags? https://ahrefs.com/blog/open-graph-meta-tags/ 

You may need to create a new image for the `og:image` property.

If you need help, let me know.

**Comment by ankman007** [comment]
> @ankman007 I have assigned this to you. I will review your PR.

Okay, thanks. 

Is there anything else I can work on? I am familiar with web dev as well.

**Comment by tamalchowdhury** [comment]
@ankman007 yes, you can propose a feature in the `issues` in GitHub. Think about what an ideal website project needs.

The site needs some text content on top of the table. Perhaps write some content (or generate using chat gpt)
